### DISCUSSION TITLE
My SMTP credentials don't work

### DISCUSSION THREAD

**Discussion by iamjohnseun** [postalserver/postal#1697] [Help with using Postal] [answered]
I have installed Postal as a Docker container on my Ubuntu VPS and everything runs smoothly, i set up my DNS records and rDNS(PTR) and all seems good. I can send emails from the Postal web interface although i am unable to access the web interface through HTTPS even after installing caddy.
My problem is, after generating credentials, my mail client is unable to authenticate with my server and send a message.
I get credentials that look like this:

`host: mail.example.com`
`username: organization/username`
`password: randomGeneratedPassword`
`port: 25`

My SMTP logs looks like this at the moment:
`smtp_1      | [9] [2021-11-10T14:54:44.288] INFO -- : Listening on  :::25`
`smtp_1      | [9] [2021-11-10T16:47:00.839] DEBUG -- : [C4GEDZ]    Connection opened from ::ffff:8.8.8.8`
`smtp_1      | [9] [2021-11-10T16:47:00.839] DEBUG -- : [C4GEDZ]    Client identified as ::ffff:8.8.8.8`
`smtp_1      | [9] [2021-11-10T16:47:00.875] DEBUG -- : [C4GEDZ] <= EHLO eu-lon-85.sample.com`
`smtp_1      | [9] [2021-11-10T16:47:00.892] DEBUG -- : [C4GEDZ] => 250-My capabilities are`
`smtp_1      | [9] [2021-11-10T16:47:00.893] DEBUG -- : [C4GEDZ] => 250 AUTH CRAM-MD5 PLAIN LOGIN`
`smtp_1      | [9] [2021-11-10T16:47:00.967] DEBUG -- : [C4GEDZ] <= MAIL FROM:<693-ICF-212.0.14577.0.0.1446.9.828653-1@eu-lon-188.sample.com>`
`smtp_1      | [9] [2021-11-10T16:47:00.967] DEBUG -- : [C4GEDZ] => 250 OK`
`smtp_1      | [9] [2021-11-10T16:47:00.984] DEBUG -- : [C4GEDZ] <= RCPT TO:<email@example.com>`
`smtp_1      | [9] [2021-11-10T16:47:01.436] DEBUG -- : [C4GEDZ] => 530 Authentication required`
`smtp_1      | [9] [2021-11-10T16:47:06.454] DEBUG -- : [C4GEDZ] <= QUIT`
`smtp_1      | [9] [2021-11-10T16:47:06.454] DEBUG -- : [C4GEDZ] => 221 Closing Connection`
`smtp_1      | [9] [2021-11-10T16:47:06.454] DEBUG -- : [C4GEDZ]    Connection closed`

I have changed some details in the log, i hope it does not confuse anyone trying to help.
I have tried different combinations but to no luck, anyone that has successfully got this to work, kindly advice.

**Comment by willpower232** [comment]
> i am unable to access the web interface though HTTPS even after installing caddy

> my mail client is unable to authenticate with my server

what error messages do you have in both cases?

**Comment by iamjohnseun** [comment]
SMTP Authentication Errors. My initial thought is that i was not using the credentials right. Perhaps there is a right way to do it.
Could you please show me the documentation or video for Postal SMTP creedential usage. That shows how the SMTP credentials be used in a mail client.
The username provided by postal looks like 
`username: organization/username`
And the password being the random generated. I am familiar with SMTP usernames being email addresses.

**Reply by willpower232** [reply]
above the credentials you should see a link to how to use the credentials, also might be worth making sure you have made an SMTP credential and not an API credential

**Reply by iamjohnseun** [reply]
The link above the credentials takes me to the page that gives me a sample credential just like the one i posted in my question. So i did try that link before coming out here to look for help. Is there an SMTP test tool you can suggest or perhaps more logs i can provide so you may get more information to help in assisting? It seems this is a problem that other people face as i also commented on #1585 and they seem to have the same problem.

1. SMTP Credentials are being generated
2. SMTP Credentials are not getting authenticated by mail clients.

Could the problem be the credentials or the server? (could be both, neither or either, idk)
Is there a video resource or tutorial on usage?
Is `organization/username` the correct way to enter the credentials on mail clients or `username@organization.tld`?

Thank you for your replies and i hope i can get help.

**Reply by JasonDevTech** [reply]
The SMTP credentials are in the following format: `mail_server_name/credential_name`

**Reply by willpower232** [reply]
I believe people use https://smtper.net/ to test, you should use the details exactly as they are shown on that page.

The results from smtper will confirm where any problems are.

**Reply by iamjohnseun** [reply]
Thank you so much for your replies. I will attempt to use https://smtper.net/ to test and provide feedback. Thank you once again.

**Reply by iamjohnseun** [reply]
I used https://smtper.net/ and i successfully received an email.

**Reply by willpower232** [reply]
so that means that wherever you are sending emails from may have it blocked.

**Reply by iamjohnseun** [reply]
I was sending from a client called "supermailer" and it did not authenticate with my server using the same details i entered in https://smtper.net. Now i just have to figure out why i cant get https on my web interface even though i installed caddy. Thank you and i appreciate all your help.

**Comment by iamjohnseun** [comment, marked as answer]
For anyone else that might need help with this,

- Use the details as generated by postal. You can get a sample credential by visiting subdomain.yourdomain.tld/org/saturn-mail-agent/servers/pan/help/outgoing
- You can test your server using https://smtper.net/
- Make sure your port is correct and you are not using a custom port. You can verify from your postal.yml file
